cli/cmd/alpha_stat.go

```diff
@@ -242,8 +259,15 @@ func metricsToRow(metrics *smimetrics.TrafficMetrics, outbound bool) []string {
 	}
 }
 
-func buildTable(outbound bool) table.Table {
+func buildTable(outbound, allNamespaces bool) table.Table {
 	columns := []table.Column{
+		table.Column{
+			Header:    "NAMESPACE",
```

Comment:
Just a heads up that this can render as:

```
❯ bin/linkerd alpha stat pods --all-namespaces
NAMESPA  NAME
linkerd  linkerd-controller-..
```

I'm not sure if `kubectl` similarly trims the column name as well.. it picks up pods in `kube-system` so `NAMESPACE` is fully rendered anyways.

Reply:
Fixed, thanks